environment.yml

```diff
@@ -7,15 +7,18 @@ channels:
   - defaults
 dependencies:
   - astropy>=5,<6
+  # frictionless 5.8 requires attrs>=22.2,
+  # but a lower version might be installed when the environment is resolved.
+  - attrs>=22.2.0
   - black
   - click=8.0.3
-  - frictionless>=5.5.0,<6
+  - frictionless>=5.8.3,<6
   - isort
   - jupytext
   - matplotlib-base>=3.5,<4
   - mergedeep>=1.3.4,<2
-  - myst-parser>=0.15,<0.17
-  - myst-nb>=0.13.0,<0.14.0
+  - myst-parser>=0.15
+  - myst-nb>=0.13.0
```

Comment:
fixing attrs apparently requires higher versions of myst

Reply:
Then we should just force a more recent version. But we should keep some upper bounds if possible.